sms_sponsorship/__manifest__.py

```diff
@@ -12,11 +12,13 @@
     'name': 'Compassion SMS Sponsorships',
     'version': '10.0.0.2.0',
     'category': 'Other',
-    'author': 'Compassion CH',
+    'author': 'Compassion CH, Odoo Community Association (OCA)',
     'license': 'AGPL-3',
     'website': 'http://www.compassion.ch',
-    'depends': ['crm_compassion', 'cms_form'],
+    'depends': ['crm_compassion', 'cms_form', 'sponsorship_compassion'],
```

Comment:
This dependency is not needed, as `crm_compassion`  already depends on `sponsorship_compassion`